rama-crypto/src/jose/jwk.rs

```diff
@@ -181,6 +217,70 @@ impl JWK {
             }
         }
     }
+
+    /// Creates a new [`JWK`] from a given [`RSAKeyPair`]
+    pub fn new_from_rsa_key_pair(rsa_key_pair: &RsaKeyPair, alg: JWA) -> Result<Self, OpaqueError> {
+        let n = rsa_key_pair.public_key().modulus();
+        let e = rsa_key_pair.public_key().exponent();
+        Ok(Self {
+            alg,
+            key_type: JWKType::RSA {
+                n: BASE64_URL_SAFE_NO_PAD.encode(n.big_endian_without_leading_zero()),
+                e: BASE64_URL_SAFE_NO_PAD.encode(e.big_endian_without_leading_zero()),
+            },
+
+            r#use: Some(JWKUse::Signature),
+            key_ops: None,
+            x5c: None,
+            x5t: None,
+            x5t_sha256: None,
+        })
+    }
+
+    fn create_public_key_sequence(n_der_encoded: Vec<u8>, e_der_encoded: Vec<u8>) -> Vec<u8> {
```

Comment:
Can you please include at the very least in the module docstring links to the RFC's which are used to implement this logic? As I assume you didn't just invent the exact algorithms used here to construct all these